### DISCUSSION TITLE
Does the Cupertino theme restrictions apply to FOSS projects ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#113] [Q&A] [answered]
Hello,

The following [FAQ](https://xel-toolkit.org/faq) statement :
> The Cupertino themes which implement macOS Human Interface Guidelines have a more restrictive license. You are allowed to use them exclusively with macOS operating system.

Is located within the following question related to commercial use.
Does that means the above-mentioned restriction only apply to commercial apps, or also to free and/or open source ones ?

Thanks

PS: your project is awesome, I've been looking for a framework like Xel since forever. Thank you for keeping it up !

**Comment by jarek-foksa** [comment, marked as answer]
The restriction applies to both commercial and free / open source apps.
